### PR TITLE
Merge Evan's vision statement

### PR DESCRIPTION
**Pull request by bobek**

### CODE REVIEW

**Comment by bobek** [comment]
Linter is failing because `opensource.com` throws 503 ;(

image

**Review by AmitKumarDas — APPROVED** [review]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,16 +1,60 @@
...
+
+We will know we are succeeding in enabling our personas when a standard user of Kubernetes can grab OpenEBS and:
+
+- *grok it* -- what it is supposed to do, how it is designed, how it is explained
+
+- and *operate it* -- knowing that it will behave in a cloud-native or container attached storage way
+
+…while the Kubernetes SREs within the Platform Team can support the use of OpenEBS at scale because they will be able to do their job. We have more work to do here in defining what these platform teams require -- thankfully we are in frequent contact with them at places such as Bloomberg, Flipkart and Optoro. 
+
+Last but certainly not least -- we are working with data and increasingly important data that needs to be secured at least for some period of time.  We must avoid losing data, understanding that solutions such as flavors of OpenEBS LocalPV themselves by design do nothing to secure the data.  Remember the old saying, when our users lose their data they typically lose their jobs.
```
Comment (kmova) [inline README.md:49]: The above needs to be rephrased or made clear that there are workloads that only need Local PV and the motivation from MayaData to work on Local PV. Inserting the data loss alongside Local PV raises the question if we should really work on Local PV at all. The other aspects of Cloud Native, as described in the many OpenEBS adoption stories indicate that teams are depending on cloud native applications to have data protection capabilities. 

Ref: 
* https://github.com/openebs/openebs/blob/master/adopters/optoro/README.md
* Cloud Native DR - https://docs.google.com/presentation/d/1GlVI8OjxNcEp3hqe-rjItTfLiuvi_RD4cTX33iyOe0A/edit#slide=id.p

Comment (bobek) [reply README.md:49]: @kmova can you please propose the wording? Alternatively, we can merge this PR and take it as followup ticket/PR. What would you prefer?

Comment (epowell101) [reply README.md:49]: I think it is clear as written that by design OpenEBS LocalPV is not designed to have replicas and secure the data.

Comment (kmova) [inline README.md:49]: @bobek - slight modification to the statement here. 

Last but certainly not least - we are working w/ data and increasingly important data that needs to be secured at least for some period of time.  We must avoid losing data, understanding that solutions such as flavors of OpenEBS LocalPV themselves by design do nothing to secure the data; _this should not be construed as an argument against LocalPV - far from it._  Remember the old saying, when our users lose their data they typically lose their jobs.

Comment (bobek) [reply README.md:49]: This looks redundant to me. We are saying, that LocalPV is designed not to protect data by itself. Like the "by design" is an important statement as we are saying, we want it this way.

I read is as -- we need to leverage strengths and be aware of limitations of particular storage engines.

**Comment by bobek** [comment]
@epowell101 resolved the original PLOW-98, thus merging.

**Pull request merged**